rules/sat/sat-inscript2.js

```diff
@@ -3,103 +3,60 @@
 
 	var satInScript2 = {
 		id: 'sat-inscript2',
-		name: 'संताली इनस्क्रिप्ट २',
+		name: 'inskript2',
```

Comment:
I have contacted some people but still unable to get  the ol chiki format of this word. So keeping it in English.